Link failed with Optimize for debugging enabled

```console
.arduino15/packages/STMicroelectronics/tools/xpack-arm-none-eabi-gcc/13.2.1-1.1/bin/../lib/gcc/arm-none-eabi/13.2.1/../../../../arm-none-eabi/bin/ld: /tmp/arduino/sketches/996241410AB91828789479A01CF30494/libraries/WiFi-ISM43362-M3G-L44/ISM43362_M3G_L44_driver.cpp.o: in function `IsmDrvClass::IsmDrvClass(SPIClass*, unsigned char, unsigned char, unsigned char, unsigned char)':
Arduino/libraries/WiFi-ISM43362-M3G-L44/src/ISM43362_M3G_L44_driver.cpp:96:(.text._ZN11IsmDrvClassC2EP8SPIClasshhhh+0x58): undefined reference to `vtable for WiFiDrvClass'
collect2: error: ld returned 1 exit status
```

--- a/src/wifi_drv.h
+++ b/src/wifi_drv.h
@@ -137,8 +137,8 @@ class WiFiDrvClass {
     virtual void ES_WIFI_getRemoteData(uint8_t sock, uint8_t *ip, uint16_t *port) = 0;
     virtual void ES_WIFI_StartServerSingleConn(uint8_t index, comm_mode mode) = 0;
     virtual void ES_WIFI_StopServerSingleConn(uint8_t index) = 0;
-    virtual void ES_WIFI_StartServerMultiConn(uint8_t socket, comm_mode mode);
-    virtual void ES_WIFI_StopServerMultiConn();
+    virtual void ES_WIFI_StartServerMultiConn(uint8_t socket, comm_mode mode) = 0;
+    virtual void ES_WIFI_StopServerMultiConn() = 0;
     virtual void ES_WIFI_StartClientConnection(uint8_t index) = 0;
     virtual void ES_WIFI_StopClientConnection(uint8_t index) = 0;
     virtual void ES_WIFI_ReceiveData(uint8_t Socket, uint8_t *pdata, uint16_t Reqlen, uint16_t *Receivedlen, uint32_t Timeout) = 0;

